src/components/VmDetail/index.js

```diff
@@ -136,7 +137,7 @@ class VmDetail extends Component {
     }
     const icon = icons.get(iconId)
     const disks = vm.get('disks')
-    const os = Selectors.getOperatingSystemByName(vm.getIn(['os', 'type']))
+    const osName = getOsHumanName(vm.get('os').get('type'))
```

Comment:
Please use `getIn()` method, it's safe with respect to `undefined`

Reply:
done